lib/runnable.js

```diff
@@ -91,14 +91,14 @@ Runnable.prototype.timeout = function (ms) {
 };
 
 /**
- * Set & get slow `ms`.
+ * Set or get slow `ms`.
  *
  * @api private
  * @param {number|string} ms
  * @return {Runnable|number} ms or Runnable instance.
  */
 Runnable.prototype.slow = function (ms) {
-  if (typeof ms === 'undefined') {
+  if (!arguments.length || typeof ms === 'undefined') {
```

Comment:
I think we can drop the `ms` check

Reply:
I've initially removed the `ms` check but tests were failing [here](https://github.com/mochajs/mocha/blob/master/test/unit/runnable.spec.js#L80).
I'm unsure if removing the ms check will introduce a breaking change.

Reply:
ugh, hum, let me think about this.

Reply:
that test is invalid and should be removed, *but* it's also a breaking change, so let's leave it for now.

Reply:
The reason why you test fails, is because when you invoke `run.slow(undefined);` `argument.length` is equal to 1. Instead of checking the number of arguments passed, you should just check if `ms` is undefined. You can do something like this:
```
if(!ms) {
  ....
}
```

Reply:
@fesebuv that's correct.  but we just want to leave as-is for now.  there's no change necessary here, since it'll be breaking.

Reply:
gotcha!